dockyard/cmd/api.py

```diff
@@ -0,0 +1,15 @@
+from wsgiref import simple_server
+
+from oslo_config import cfg
+from oslo_log import log as logging
```

Comment:
Currently application is running in foreground, we need to make it run as a service.